rustls/src/client/client_conn.rs

```diff
@@ -139,7 +139,7 @@ impl IdentityRequest<'_> {
     /// `x509-parser`.
     ///
     /// [`DistinguishedName`]: crate::DistinguishedName
-    pub fn root_hint_subjects(&self) -> &[&[u8]] {
+    pub fn root_hint_subjects(&self) -> &[DistinguishedName] {
```

Comment:
I would reiterate in the doc comment here that this type does not validate or guarantee the encoding.

Reply:
Done. Feel free to nitpick my docs.